backend/read_tls.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func PrepareTLSCfg(certPath string, rootPath string, caPath string) (*tls.Config, error) {
-	tlsConfig := &tls.Config{InsecureSkipVerify: true}
+	tlsConfig := &tls.Config{InsecureSkipVerify: false}
```

Comment:
With the self-signed certificates, that must be configurable

Reply:
Yes, it's reasonable. I turned it on and created task for implementing checkbox to configure this